spec/services/valid_test_data_generators/applications_populater_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ValidTestDataGenerators::ApplicationsPopulater do
+  let(:lead_provider) { create(:lead_provider) }
+  let(:cohort) { create(:cohort) }
+
+  before { allow(Rails).to receive(:env) { environment.inquiry } }
+
+  subject { described_class.new(lead_provider:, cohort:, number_of_participants: 50) }
```

Comment:
for testing purposes could we set this to 2 to avoid creating too many and slowing things down?